Add the Contribution Guide and Review Community and Security Pages (#177)
ballerina-platform/ballerina-dev-website: community.md

The Ballerina community has been bootstrapped by <a href="https://wso2.com">WSO2</a> via sponsorship of the design & implementation of the Ballerina platform. Ballerina is a fully open source project with all source code licensed under the <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a> and the specifications licensed under the <a href="https://creativecommons.org/licenses/by-nd/4.0">Creative Commons Attribution-NoDerivatives 4.0 International</a> license.

We warmly welcome all contributors to the community to help establish Ballerina as a truly community-owned resource!

> ## How to get help

Reviewer: How about "Need help?" instead?

Reviewer 2: +1 for @pubudu91's suggestion.

Author: Changed to "Seek for help".

The Ballerina community has been bootstrapped by <a href="https://wso2.com">WSO2</a> via sponsorship of the design & implementation of the Ballerina platform. Ballerina is a fully open source project with all source code licensed under the <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a> and the specifications licensed under the <a href="https://creativecommons.org/licenses/by-nd/4.0">Creative Commons Attribution-NoDerivatives 4.0 International</a> license.

We warmly welcome all contributors to the community to help establish Ballerina as a truly community-owned resource!

- [Try out Ballerina](#try-out-ballerina)
- [Seek for help](#seek-for-help)
- [Report issues](#report-issues)
- [Help us grow](#help-us-grow)
- [Contribute to Ballerina](#contribute-to-ballerina)

## Try out Ballerina

Use the below resources to learn Ballerina and try out writing Ballerina code on your own.
- [Quick Tour](https://ballerina.io/learn/quick-tour)
- [Ballerina By Examples](https://ballerina.io/learn/by-example/)

> - [How-To Guides](https://ballerina.io/learn/)

Reviewer 3: This links to learn page. It seems we don't have a direct link to How to guides.

Author: We don't have a direct link to How-To Guides. Please correct me if I am wrong.

Reviewer 3: In that case, should we reword this to go through learn page entirely, rather than each section?

Author: Added a single link to the Learn page.

- [The Standard Library](https://ballerina.io/learn/api-docs/ballerina/)

> >**Tip:** Do not forget to share the wealth by publishing your Ballerina module on <a href="https://central.ballerina.io">Ballerina Central</a> so that the whole community can benefit from your work. 

Reviewer 3: This content seems repeating in the `Help us grow` section as well.

Author: Removed this Tip.

<input class="cTextfieldstyle" maxlength="90" value="" id="emailUserOS" name="email" placeholder="I consent to join the email list" title="Email" type="text">
</form> -->

> ## How you can help 

Reviewer: Shall we try and keep these headings catchy/lively? How about something like "Help us grow" or "We need your help!"

Author: Changed as follows:
- Seek for help
- Help us grow

Getting a language established is like getting a new religion going. We really need you, our fans, to help spread the word.

> - Start with starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!

Reviewer 3: Instead of main repo, shall we rephrase it as Ballerina Github repo?

Reviewer 4: Shouldn't this be
```suggestion
- Start by starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!
```

Author: > Instead of main repo, shall we rephrase it as Ballerina Github repo?

Done

- Follow us on Twitter: <a href="https://twitter.com/ballerinalang">@ballerinalang</a>. Tweet with the "#ballerinalang" hashtag.
- Share the wealth by publishing your Ballerina module on <a href="https://central.ballerina.io">Ballerina Central</a>. 

> - Give us a buzz on <a href="mailto:contact@ballerina.io">contact@ballerina.io</a> if you want to organize a local meetup or hackathon. WSO2 will get right on it and help with presentation/training content, logistics, swag, and some funds for munchies.

Reviewer 4: How about
```suggestion
- Buzz us on <a href="mailto:contact@ballerina.io">contact@ballerina.io</a> if you want to organize a local meetup or hackathon. WSO2 will get right on it and help with presentation/training content, logistics, swag, and some funds for munchies.
```

> - Report an issue/bug in the below GitHub repos and tell us if you hit a bump on the road. We want to fix all bugs!

Reviewer 3: What is the reason for merging the reporting issues section and helping us sections? IMO, reporting issues section should be highlighted first. That will give the idea of the basic flow - where we help the users first by resolving their issues first and then they help us by starring and tweeting after they are happy with the experience.

Reviewer: How about something like the following for the complete flow? 
1. Invite the users to try out Ballerina (maybe we can point them to the quick start guide, BBEs, how-to guides and API docs) and develop something.
2. Ask them to create issues for any issues/bugs they find in the process.
3. Ask them to publish their work to Ballerina Central once they are ready so that others can benefit from their work.
4. Ask for their help if they liked the experience so far. e.g., starring the repo, following on Twitter, organizing local meetups etc.

Reviewer 4: +1 for the flow @pubudu91 has suggested.

Author: Added the below sections:
- Try out Ballerina
- Seek for help
- Report issues
- Help us grow
- Contribute to Ballerina

   - compiler, runtime, standard library, or tooling: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
   - language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
   - website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 
   - security flaw: send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.

> - Join with us and contribute to the source code to make Ballerina better! To be aware of the ground rules as you start, see the <a href="https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md">Contributor Guide</a>. Happy contributing! 

Reviewer 4: Do we need "with"? Can't we just say

```suggestion
- Join us and contribute to the source code to make Ballerina better! To be aware of the ground rules as you start, see the <a href="https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md">Contributor Guide</a>. Happy contributing! 
```

   </div>
</div>

## Report issues

> Report an issue/bug in the below GitHub repos and tell us if you hit a bump on the road. We want to fix all bugs!

Reviewer: ```suggestion
Hit a bump on the road? Report an issue in the relevant repo out of the GitHub repos listed below. We want to fix all bugs!
```

   - compiler, runtime, standard library, or tooling: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
   - language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
   - website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 

>    - security flaw: send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.

Reviewer 3: Shouldn't we start with an upper case letter in each bullet point?

Author: Done.

## Help us grow 

Getting a language established is like getting a new religion going. If you like the experience so far, help us to spread the word.

> - Start by starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!

Reviewer 3: It seems the comment [1] is not addressed even though it is marked as resolved. 

[1] https://github.com/ballerina-platform/ballerina-dev-website/pull/177#discussion_r363678508

Author: Done.

<input class="cTextfieldstyle" maxlength="90" value="" id="emailUserOS" name="email" placeholder="I consent to join the email list" title="Email" type="text">
</form> -->

## Contribute to Ballerina

> Join us and contribute to the source code to make Ballerina better! To be aware of the ground rules as you start, see the <a href="https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md">Contributor Guide</a>. Happy contributing! 

Reviewer 3: Since we are adding a contribution guide page for the b.io along with this PR, shall we link that instead of contributing guide of lang repo?

Author: Changed the link to "<a href="https://ballerina.io/contribution-guide">Contribution Guide</a>".
